docs/B-Computations/testing-and-debugging.md

```diff
@@ -60,7 +64,7 @@ Semantic errors are errors that fail to implement the intent and meaning of the
 Techniques for identifying semantic errors include:
 
 * vocalization - use your sense of hearing to identify the error \(compound conditions\)
-* intermediate output - printf\(\) statements at critical stages
+* intermediate output - `printf()` statements at critical stages
 * walkthrough table
 * interactive debugging using
   * Visual Studio IDE - integrated debugger for Windows OSs
```

Comment:
![image](https://user-images.githubusercontent.com/44437742/142304420-3ccb046f-5408-4767-a8c6-cb00825f2d08.png)
I think you should add a white space before line 71 and 72 for consistency

Reply:
<img width="699" alt="image" src="https://user-images.githubusercontent.com/55283735/142343230-e932071e-aad3-479b-ba08-f1b5e6fa600e.png">

On local machine it looks consistent.